tests/e2e/test_compilation.py

```diff
@@ -139,9 +139,7 @@ def test_invalid_query_dbt_compilation_error(self):
         # Compile a query which results in a dbt compilation error
         invalid_query = "select * from {{ ref('not_a_model') }}"
         resp = self.compile_against_state(self.state_id, invalid_query)
-        data = resp.json()
         self.assertEqual(resp.status_code, 400)
-        assert bool(re.match("compilation error", data["message"], re.I))
```

Comment:
The error message text that core produces has changed between 1.5 and 1.6. Afaict, there's no behavior driven off of the error message, so I have opted to remove these checks from the test since they are implementation.

Reply:
No, we shouldn't be doing anything of the sort

Reply:
Perfecto, thanks for the confirmation 👍